ecs-deploy/ecs_service.tf

```diff
@@ -42,6 +42,19 @@ data "aws_subnets" "public" {
   }
 }
 
+data "aws_subnets" "private" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.main.id]
+  }
+  filter {
+    name = "tag:Name"
+    values = [
+      "${var.project}-${var.environment}-private-*",
```

Comment:
NIT: should we just have `data "aws_subnets" "main" { ... }` and use  `"${var.project}-${var.environment}-${var.subnet_type}-*"`?

Reply:
Good suggestion. 👍 
I have updated the codes